config/crds/workload.kcp.dev_synctargets.yaml

```diff
@@ -29,6 +29,10 @@ spec:
       name: Synced API resources
       priority: 3
       type: string
+    - jsonPath: .metadata.labels['internal\.workload\.kcp\.dev/key']
+      name: Key
+      priority: 4
```

Comment:
has this changed that non-priority zero column are dropped? @davidfestal do you know?

Reply:
if you tried it with current kubectl, then it is fine and somebody fixed it.

Reply:
kubectl "-o wide" displays the columns.

Reply:
but not without wide?

Reply:
nope... using latest kubectl